python/ray/data/tests/test_transform_pyarrow.py

```diff
@@ -181,6 +185,32 @@ def test_arrow_concat_tensor_extension_uniform_but_different():
     # fails for this case.
 
 
+@pytest.mark.skipif(
```

Comment:
I'm not sure if these actually get run in CI if I put the skipif and require PyArrow version >= 9.0.0. Can someone check?

Reply:
yes. it got run here https://buildkite.com/ray-project/microcheck/builds/2214